### DISCUSSION TITLE
How to increase "preview" image in "Publication"

### DISCUSSION THREAD

**Discussion by awsaf49** [alshedivat/al-folio#1577] [Q&A] [answered]
## Current Behavior

This is an extension of this issue (#1552). Currently, the preview image looks like this,

![image](https://github.com/alshedivat/al-folio/assets/36858976/9a93aaa0-9b36-4f93-be5a-ab129b0e591c)

## Expected Result
I want to make the "preview" image large so that it may be something like this,

1. [Site1](https://shivamduggal4.github.io/) 
![image](https://github.com/alshedivat/al-folio/assets/36858976/bf5eb78d-91f7-46b7-b346-f8dd95b78154)

2. [Site 2](https://shagunuppal.github.io/) 
![image](https://github.com/alshedivat/al-folio/assets/36858976/66cd641b-a3e0-4052-9af3-38f131540b00)

## Previous Attempt
I tried to change the `col-sm-2` in `bib.html` but it creates a misalignment,
![](https://user-images.githubusercontent.com/36858976/253366520-67addd57-3c2c-4f91-b74a-c8ec69fd2b79.png)

> Your help is appreciated

**Comment by awsaf49** [comment]
cc: @alshedivat

**Comment by riasat-sheikh** [comment]
Try changing the column size in the `_layotus/bib.html` file. Current preview part uses a `col-sm-2` you can change it to `col-sm-3` or more as per your need. But do remember to change the second column as well. If I remember correctly, total sum should result to 12.

```html
// other codes

   <div class="col-sm-3 {% if entry.preview %}preview{% else %}abbr{% endif %}">

// other codes

<div id="{{entry.key}}" class="col-sm-8">

// other codes
```

**Comment by awsaf49** [comment]
@TheFermi0n is the `<div id="{{entry.key}}" class="col-sm-8">` second column?

Currently, the total sum is `(2 + 8) = 10`, If I change the first column to `3` doesn't it mean the second column will be `10 - 3 = 7` ??

**Comment by awsaf49** [comment]
Btw I tried and it creates misalignment like this,
![253366520-67addd57-3c2c-4f91-b74a-c8ec69fd2b79](https://github.com/alshedivat/al-folio/assets/36858976/669fc8db-d1d6-4f00-ba53-0502a6264637)

**Comment by awsaf49** [comment]
I think after increasing columns somehow it isn't affecting the preview image size

```html
      <div class="row">
......
          <img data-zoomable class="preview z-depth-1 rounded" src="{{ entry.preview }}" width="500">
```

**Comment by riasat-sheikh** [comment, marked as answer]
> is the `<div id="{{entry.key}}" class="col-sm-8">` second column?
> 
> Currently, the total sum is `(2 + 8) = 10`, If I change the first column to `3` doesn't it mean the second column will be `10 - 3 = 7` ??

Yes that is the second column. For al-folio it is 10, yes. So if you change your first column to 3 then you still have 1 unit left so no worries with the second column.

Additionally, for the misalignment what you can do is play with the `_sass/base.scss` file and change this part accordingly

```css
.preview {
        width: 100%;
        min-width: 80px;
        max-width: 200px;
      }
```

specifically increasing the max-width might fix the misalignment.

**Comment by awsaf49** [comment]
It worked thanks @TheFermi0n :D
